tests/linearGaussian_npse_test.py

```diff
@@ -157,20 +158,22 @@ def simulator(theta):
     check_c2st(samples, target_samples, alg="npse_different_dims_and_resume_training")
 
 
-@pytest.mark.xfail(
-    reason="iid_bridge not working.",
-    raises=AssertionError,
-    strict=True,
-    match="Score accumulation*",
-)
-@pytest.mark.parametrize("num_trials", [2, 10])
-def test_npse_iid_inference(num_trials):
+@pytest.mark.slow
+@pytest.mark.parametrize("sde_type", ["vp", "ve", "subvp"])
+@pytest.mark.parametrize("with_improper_prior", [False, True])
+def test_npse_iid_inference(sde_type, with_improper_prior):
     """Test whether NPSE infers well a simple example with available ground truth."""
 
     num_dim = 2
-    x_o = zeros(num_trials, num_dim)
     num_samples = 1000
     num_simulations = 3000
+    iid_methods = ["fnpe", "gauss", "auto_gauss", "jac_gauss"]
+    num_good_evals_per_method = {
```

Comment:
please rename to `num_trials` and use `pytest.param` to have specific num_trials per method.

Reply:
But the think here is that I also do not want to retrain the score_estimator all the time. If I do it with parameters or fixtures, will it rerun the whole test?